packages/node_modules/@ciscospark/widget-space/src/enhancers/errors.js

```diff
@@ -90,6 +93,18 @@ function checkForErrors(props) {
       });
     }
   }
+
+  const disabledInitialErrorId = 'ciscospark.container.space.error.disabledInitial';
+  activityTypes.forEach((activity) => {
+    if (activity.name === initialActivity && spaceActivities && spaceActivities[activity.name] === false
+    && !errors.get('errors').has(disabledInitialErrorId)) {
+      props.addError({
+        id: disabledInitialErrorId,
+        displayTitle: formatMessage(messages.disabledInitialActivity),
+        temporary: false
+      });
+    }
+  });
```

Comment:
Are we sure we want to to acknowledge that the activity is disabled? I would think if the developer has purposely disabled the activity, we should say the activity provided is not valid.
I'm drawing a comparison from if you're not logged in and try to hit a private URL on github, it gives you 404 instead of a permission denied error.

Reply:
Ok, that works. So to make sure I understand what you mean -- just reword the error message?

Reply:
Let me know if this works for you instead @bzang: 'Error: The selected initial activity is invalid'

Reply:
Yea, I think it would make sense to change the wording, but also handle the case where if the initialActivity doesn't exist in our activity list we also pop the same message.